war/pom.xml

```diff
@@ -38,13 +38,13 @@
   interface) and the SlipStreamConnector module.
 -->
 
-<!--
     <dependency>
       <groupId>com.sixsq.slipstream</groupId>
       <artifactId>SlipStreamConnector-CloudStack-jar</artifactId>
       <version>${project.version}</version>
     </dependency>
 
+<!--
```

Comment:
@st 
Did you un-outcomment (?!) the CloudStack connector `<dependency>` in the `war/pom.xml` on purpose?